webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala

```diff
@@ -893,14 +893,10 @@ class ComplianceAPIService(
       allDirectives:         Map[DirectiveId, (FullActiveTechnique, Directive)],
       nodeFacts:             MapView[NodeId, CoreNodeFact],
       nodeSettings:          Map[NodeId, RudderSettings],
```

Comment:
We have two source of the same elements, this is often a by idea because most of the time, it comes with hidden assumption about what properties hold on both group to not break the following implementation logic